solana/pyth2wormhole/client/tests/fixtures/pyth.rs

```diff
@@ -1,11 +1,10 @@
 //! This module contains test fixtures for instantiating plausible
 //! Pyth accounts for testing purposes.
 use {
-    pyth_client::{
-        AccKey,
+    pyth_sdk_solana::state::{
         AccountType,
-        Price,
-        Product,
+        PriceAccount,
```

Comment:
I'm not sure that I understand clippy's reasoning here. Hos does it make this decision?

Reply:
Ohhh, I get it now, these are the same structs but in the changed package. I think that's alright!